charts/seed-operatingsystemconfig/original/values.yaml

```diff
@@ -59,6 +59,22 @@ worker:
       imageFSInodesFree: 0Mi
       nodeFSAvailable: 0Mi
       nodeFSInodesFree: 0Mi
+    kubeReserved:
+      cpu: 80m
+      memory: 1Gi
+      # pid: 1000
+      # ephemeral-storage: 10Gi
+    # systemReserved:
+      # cpu: 80m
+      # memory: 1Gi
+      # pid: 1000
+      # ephemeral-storage: 10Gi
+    enforceNodeAllocatable:
+      - pods
+      # - kube-reserved
+      # - system-reserved
+    # kubeReservedCgroup: runtime.slice
```

Comment:
Gardener does not (yet?) create the cGroups according to the [recommendation made here](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/node-allocatable.md#recommended-cgroups-setup) which would expect a cGroup named "runtime.slice". Instead all the systemd services installed by gardener are currently under system.slice cGroup.

Having this as a default comment here is a problem I think, because when enabled might lead to the kubelet to fail (when allocatable is also enabled for kube-reserved).